torch/_inductor/kernel/flex_attention.py

```diff
@@ -740,6 +741,106 @@ def flex_attention(
         SPARSE_KV_BLOCK_SIZE,
         mask_graph,
     ) = block_mask
+    if query.get_device().type == "cpu":
+        placeholder_inps = [
+            create_placeholder(name, dtype, query.get_device())
+            for name, dtype in [
+                ("arg0_1", query.get_dtype()),
+                ("arg1_1", torch.int32),
+                ("arg2_1", torch.int32),
+                ("arg3_1", torch.int32),
+                ("arg10_1", torch.int32), # TODO: fix the random picked name here: arg10_1
```

Comment:
Can we align these names with cuda?

Reply:
Yes, we checked the buffers and names, and will revise them.